out/production/bpdm-certificate-management/META-INF.spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1 @@
+org.eclipse.tractusx.bpdm.common.config.BpdmCommonAutoConfiguration
```

Comment:
Why is there a autoconfiguration here? This belongs to spring libraries and not applications

Reply:
Removed